app/controllers/ExperimentStepsController.php

```diff
@@ -0,0 +1,133 @@
+<?php
+/**
+ * app/controllers/ExperimentStepsController.php
+ *
+ * @author Nicolas CARPi <nicolas.carpi@curie.fr>
+ * @copyright 2012 Nicolas CARPi
+ * @see https://www.elabftw.net Official website
+ * @license AGPL-3.0
+ * @package elabftw
+ */
+namespace Elabftw\Elabftw;
+
+use Exception;
+
+/**
+ *
```

Comment:
add a description of this file here. It shows up in the API doc that you can generate with `grunt api`.